app/test_main.py

```diff
@@ -1,3 +1,27 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_result",
+    [
+        (0, 0, [0, 0]),
+        (10, 10, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (20, 20, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (30, 30, [3, 3]),
+        (100, 100, [21, 17]),
```

Comment:
The expected results for the test cases seem to follow a specific pattern. However, without knowing the logic of `get_human_age`, it's unclear if these expected results are correct. Double-check the logic in `get_human_age` to ensure these expected results are accurate.